telecine/build.gradle

```diff
@@ -72,14 +72,14 @@ android {
 }
 
 dependencies {
-  compile 'com.android.support:support-annotations:23.2.1'
+  compile 'com.android.support:support-annotations:23.4.0'
```

Comment:
These changes are not related to build tools and android plugin and I would prefer they were done separately.